lib/fortnox/api/repositories/base.rb

```diff
@@ -100,11 +103,90 @@ def config
           Fortnox::API.config
         end
 
-        def token_store_not_found!(store_name)
-          raise MissingConfiguration,
-                "There is no token store named #{store_name}. Available stores are #{config.token_store.keys}."
+        def get_token_store(name)
+          storage = config.storages&.dig(name.to_sym) || config.storage
+          validate_store(storage, name)
+        end
+
+        def validate_store(store, name)
```

Comment:
I'd say no ... This is defensive programming at its worst. The store will throw if we try to access a `name` that it doesn't have. If you want to give a nicer message you can catch that error and reformat it there.

We have given an API that the developers of stores have to adhere to, if you want more control you should make an interface that they have to implement, something like:
```
class Fortnox::API::TokenStore
  class MissingImplementation < StandardError; end
  def access_token(name = :default)
    raise MissingImplementation, "TokenStore must implement access_token="
  end
  ...
end
```